src/nidigital/templates/session.py/fancy_fetch_capture_waveform.py.mako

```diff
@@ -34,15 +34,16 @@
         import collections
         import sys
```

Comment:
Can't leave a comment on line 3
`    '''Dispatches to the appropriate "fetch waveform into" method based on the waveform type.'''`

That seems wrong.